tests/kuttl/cloudscale-status-test/00-assert-cloudscale-composition.yaml

```diff
@@ -1,6 +1,6 @@
----
 apiVersion: kuttl.dev/v1beta1
 kind: TestAssert
+timeout: 300
```

Comment:
I would leave timeout to 60s and in all subsequent tests.

Reply:
I tried with 60s, 100s and 120s. The first two assertions only worked with 120s and the last one with 60s is ok.